tests/nested/core/core-snap-plug-panic/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: core revert test to regression test panic SNAPDENG-8452
+
+systems: [ubuntu-16.04-64]
+
+kill-timeout: 60m
```

Comment:
it is so much I think. I would use the default one. In the tests los I see it takes less than 10 minutes to run